fluentui_persona/src/main/java/com/microsoft/fluentui/persona/PersonaView.kt

```diff
@@ -151,6 +162,7 @@ class PersonaView : ListItemView {
         avatarView.avatarImageBitmap = avatarImageBitmap
         avatarView.avatarImageUri = avatarImageUri
         avatarView.avatarBackgroundColor = avatarBackgroundColor
+        avatarView.avatarContentDescriptionLabel  = avatarContentDescriptionLabel
```

Comment:
(nit: extra space before =)

Reply:
Thanks for pointing these outs. My laptop's keyboard is giving some issues while entering spaces and hence these issues.